features/enlinkd/tests/src/test/java/org/opennms/netmgt/nb/Nms10115NetworkBuilder.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to The OpenNMS Group, Inc (TOG) under one or more
+ * contributor license agreements.  See the LICENSE.md file
+ * distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * TOG licenses this file to You under the GNU Affero General
+ * Public License Version 3 (the "License") or (at your option)
+ * any later version.  You may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at:
+ *
+ *      https://www.gnu.org/licenses/agpl-3.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ */
+package org.opennms.netmgt.nb;
+
+import org.opennms.core.utils.InetAddressUtils;
+import org.opennms.netmgt.model.OnmsNode;
+
+import java.net.InetAddress;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author <a href="mailto:antonio@opennms.it">Antonio Russo</a>
+ */
+
+public class Nms10115NetworkBuilder extends NmsNetworkBuilder {
```

Comment:
Just out of curiosity, why is this named Nms10115NetworkBuilder and not Nms18541NetworkBuilder? The issue https://opennms.atlassian.net/browse/NMS-10115 is something totally different.

Reply:
is the address of the network devices...10.115.180.*
I got the first numbers...

Reply:
This is interesting. It could be my next choise.